node/src/utils/round_robin.rs

```diff
@@ -43,6 +43,9 @@ pub struct WeightedRoundRobin<I, K> {
 
     /// Whether or not the queue is sealed (not accepting any more items).
     sealed: AtomicBool,
+
+    /// Only dump event counts when there are more of them than in the previous report.
```

Comment:
I think it's worth commenting that event counts are only dumped if this member variable is `Some`.

Reply:
Comment updated in https://github.com/casper-network/casper-node/pull/4295/commits/46ac95c0df9ad2b05c63d22b7ffa959bf655327f